stix_shifter/stix_translation/src/modules/security_advisor/json/to_stix_map.json

```diff
@@ -0,0 +1,80 @@
+{
+
+  "author_id": {
+    "key": "author.user_id"
+  },
+
+  "author_account_id": {
+    "key": "author.author_account_id"
+  },
+
+  "author_email": {
+    "key": "author.email_id"
+  },
+
+  "context_resource_name": {
+    "key": "context.context_resource_name",
+    "cybox": false
+  },
+  "description": {
+    "key": "finding.description",
```

Comment:
The convention for custom objects is to start the name with an x_ So in this case it would be `x_finding.description`.

Reply:
Done